src/app/WriteClient.h

```diff
@@ -52,22 +56,21 @@ class WriteClient : public Messaging::ExchangeDelegate
      */
     void Shutdown();
 
-    /**
-     *  Once SendWriteRequest returns successfully, the WriteClient will
-     *  handle calling Shutdown on itself once it decides it's done with waiting
-     *  for a response (i.e. times out or gets a response).
-     *  If SendWriteRequest is never called, or the call fails, the API
-     *  consumer is responsible for calling Shutdown on the WriteClient.
-     */
-    CHIP_ERROR SendWriteRequest(NodeId aNodeId, FabricIndex aFabricIndex, SecureSessionHandle * apSecureSession);
-
     CHIP_ERROR PrepareAttribute(const AttributePathParams & attributePathParams);
     CHIP_ERROR FinishAttribute();
     TLV::TLVWriter * GetAttributeDataElementTLVWriter();
 
+    uint64_t GetAppIdentifier() const { return mAppIdentifier; }
```

Comment:
App identifier seems to be used as a sequence number. Could we name this as 'SequenceNumber' to make variable naming and usage consistent?

Consistency-wise, please also either use 'App' or 'Application', but not both. Currently Get uses 'App' and Set uses 'Application'.

Reply:
The AppIdentifier will be used by end applications, and not necessarily be a sequence number, per design, we will use delegate + app id to provide more flexible interface at the end of day.

Changed all to {Get|Set}AppIdentifier.